src/pepr/operator/controllers/keycloak/client-sync.ts

```diff
@@ -58,7 +58,13 @@ export async function keycloak(pkg: UDSPackage) {
     clients.set(client.clientId, client);
   }
 
-  await purgeSSOClients(pkg, [...clients.keys()]);
+  // Purge orphaned clients
+  try {
+    await purgeSSOClients(pkg, [...clients.keys()]);
+  } catch (e) {
+    log.error(e, `Failed to purge orphaned clients in for ${pkg.metadata!.name!}: ${e}`);
+  }
```

Comment:
Just to note, this maintains the previous behavior where failures would just log out but we could also throw an error here or make an event. I made this change because `purgeSSOClients` will throw an error now if a token can't be found in the store.